### DISCUSSION TITLE
Year End Procedure

### DISCUSSION THREAD

**Discussion by TonySLondon** [moneymanagerex/moneymanagerex#5258] [General]
Normally with accounts programs and in manual accounting, a year end closure occurs and balances are transferred to the new year. I have suggested this before but is 'sniffed at'.  This make moneymanagerex a less than full accounts program. I do not ant an ongoing database the fills and fills; this is NOT solid accounting procedure.

**Comment by vomikan** [comment]
For example, I personally do not transfer any balances from year to year. I have observed this only at enterprises.
I agree that for greater peace of mind, you can sum up the results of the year and close the data from changes.
There is an option for this in the account properties.

Related issues.
https://github.com/moneymanagerex/moneymanagerex/issues/1297
https://github.com/moneymanagerex/moneymanagerex/issues/2257

**Comment by tactilis** [comment]
> Normally with accounts programs and in manual accounting, a year end closure occurs and balances are transferred to the new year.

Yes, that is _one_ way of doing things.

But it's not the only way.


> I have suggested this before but is 'sniffed at'. This make moneymanagerex a less than full accounts program.

No it does not. It is a fact that MMEX works perfectly for many users. I have never had a wish to perform an end of year closure. I use the tools in MMEX to slice and dice my accounts into whatever periods I want to see.

> I do not ant an ongoing database the fills and fills; this is NOT solid accounting procedure.

Assuming you meant:
> I do not _want_ an ongoing database _that_ fills and fills; this is NOT solid accounting procedure.

Then, respectfully, may I suggest that MMEX is probably not the best tool for you. You may be better served by one of the many other accounting applications that are available.

**Comment by whalley** [comment]
As mentioned above, the feature has been asked for but has not yet been implemented. I guess MMEX is typically used as a *personal* finance application where these practices do not occur (and people want to see all the history in one go) so that would affect its prioritisation
